core/services/flux_monitor.go

```diff
@@ -236,6 +243,51 @@ func (f pollingDeviationCheckerFactory) New(initr models.Initiator, runManager R
 	)
 }
 
+// ExtractFeedURLs extracts a list of url.URLs from the feeds parameter of the initiator params
+func ExtractFeedURLs(initr models.InitiatorParams, orm *orm.ORM) ([]*url.URL, error) {
```

Comment:
Why operate on the InitiatorParams instead of the Feeds type?

Reply:
there is no longer a Feeds type... I could probably make this operate on `[]interface{}` but that might be more confusing. Alternatively I could create a Feeds type and set it to `[]interface{}`. Thoughts?

Reply:
I believe its possible but it may not be worth the effort given current priorities.  We would type alias JSON to Feeds though and associate some of the methods with it.  Would that be easy?

Reply:
hmmm I didn't think about that. That might be a winning idea! I'll play around with it.

Reply:
Quick note, use 'type Feeds = models.JSON' instead of go's alias.  The alias keyword doesn't fit this case ;)